.github/ISSUE_TEMPLATE/security.yml

```diff
@@ -1,34 +1,41 @@
-name: '⚠️ Security&Privacy issue'
```

Comment:
I ended up removing this file in the later commits because I figured you would add it yourself.